.strict-typing

```diff
@@ -79,9 +79,9 @@ homeassistant.components.button.*
 homeassistant.components.calendar.*
 homeassistant.components.camera.*
 homeassistant.components.canary.*
-homeassistant.components.cover.*
```

Comment:
It looks like there are unrelated changes in this file

Reply:
The sort order of the file changed.

Reply:
Please keep the sort order the same as the branch this it targeting since https://github.com/home-assistant/core/pull/81331 has not merged